vk_video_decoder/libs/VkVideoDecoder/VkVideoDecoder.cpp

```diff
@@ -208,7 +208,14 @@ int32_t VkVideoDecoder::StartVideoSequence(VkParserDetectedVideoFormat* pVideoFo
     }
 
     m_capabilityFlags = videoDecodeCapabilities.flags;
+    if (IsDstDpbDistinctImages()) {
+        m_useSeparateOutputImages = true;
+    }
 
```

Comment:
We still nee the useLinearOutput case here:

if (IsDstDpbDistinctImages() || useLinearOutput ) {